<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logDialog</class>
 <widget class="QWidget" name="logDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="lw_log">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <property name="animated">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cb_time">
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_sockets">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_type">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>All</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Arp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TCP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UDP</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clear">
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="icon">
        <iconset resource="../../netemul.qrc">
         <normaloff>:/im/images/clean.png</normaloff>:/im/images/clean.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../netemul.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>lw_log</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
